src/main/java/com/training/dependencyInjection/listAlgorithms/AppConfig.java

```diff
@@ -0,0 +1,19 @@
+package com.training.dependencyInjection.listAlgorithms;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Primary;
+
+@Configuration
+public class AppConfig {
+    @Bean
+    public AscendingSort ascendingSort() {
```

Comment:
Instead of using @Bean here, can you use @Component in the AscendingSort and DecendingSort. Also you can use @Quilfier

Reply:
Well, the changes have been committed.
Hope you check the `remove appConfig` commit.

Reply:
Approved